src/utils/fake-backend.js

```diff
@@ -1,4 +1,19 @@
 // array in local storage for registered users
+
+let usersArray = [
+  {
+    firstName: 'ahmed',
+    lastName: 'aly',
+    username: 'ahmed.aly',
+    password: 'umbrage',
```

Comment:
You can set this in local storage in your browser manually but this works too. We’ll end up ripping this out later anyway so this is good for now